### PR TITLE
[CLOUDFRONT-7696] eslint fix

### PR DESCRIPTION
**Pull request by nanov94**

### CODE REVIEW

File: src/services/metadata.ts [hunk 1]
```diff
@@ -79,10 +79,14 @@ function getUpdatedMetadataString(newMetadatas: string[], defaultMetadata = ''):
     return `${metadataBorder}${updatedMetadata}${metadataBorder}`;
 }
 
-async function getUpdatedMetadata(result: any, options: MetaDataOptions, fileContent: string): Promise<Record<string, any>> {
+async function getUpdatedMetadata(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    result: any,
```
Comment (vovcyan) [inline src/services/metadata.ts:84]: Do we really need to disable eslint in this case?
Let's use `unknown` type.

File: src/vcs-connector/connector-models.ts [hunk 2]
```diff
@@ -7,7 +7,9 @@ import {
 
 /* eslint-disable camelcase */
 export interface ConnectorValidatorProps {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     validateFn: (value: any) => Boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (vovcyan) [inline src/vcs-connector/connector-models.ts:12]: Same

File: src/vcs-connector/connector-validator.ts [hunk 3]
```diff
@@ -65,6 +66,7 @@ export function validateConnectorFields(
     }
 
     let isValidProperties = true;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (vovcyan) [inline src/vcs-connector/connector-validator.ts:69]: Same

File: src/vcs-connector/connector-validator.ts [hunk 4]
```diff
@@ -48,6 +48,7 @@ function notEmptyValue(value: string | undefined): boolean {
 export function validateConnectorFields(
     sourceType: SourceType,
     fieldNames: GitHubConnectorFields[],
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (vovcyan) [inline src/vcs-connector/connector-validator.ts:51]: Same

File: src/vcs-connector/connector-models.ts [hunk 5]
```diff
@@ -7,8 +7,9 @@ import {
 
 /* eslint-disable camelcase */
 export interface ConnectorValidatorProps {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (vovcyan) [inline src/vcs-connector/connector-models.ts:10]: unknown?

**Review by vovcyan — APPROVED** [review]

File: src/vcs-connector/connector-models.ts (hunk 5 above)
Comment (nanov94) [reply src/vcs-connector/connector-models.ts]: **DONE**

File: src/vcs-connector/connector-models.ts (hunk 2 above)
Comment (nanov94) [reply src/vcs-connector/connector-models.ts]: **DONE**

File: src/vcs-connector/connector-validator.ts (hunk 3 above)
Comment (nanov94) [reply src/vcs-connector/connector-validator.ts]: **DONE**

File: src/vcs-connector/connector-validator.ts (hunk 4 above)
Comment (nanov94) [reply src/vcs-connector/connector-validator.ts]: **DONE**

File: src/services/metadata.ts (hunk 1 above)
Comment (nanov94) [reply src/services/metadata.ts]: **DONE**

**Pull request merged**